monitoring/node_collector.go

```diff
@@ -0,0 +1,77 @@
+package monitoring
+
+import (
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/longhorn/longhorn-manager/datastore"
+	"github.com/longhorn/longhorn-manager/types"
+
+	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta1"
+)
+
+type NodeCollector struct {
+	*baseCollector
+	newNodeStatusMetric metricInfo
```

Comment:
Why is this called `newNodeStatusMetric`?
could it be called just `statusMetric` since it's private and inside of the NodeCollector?

Reply:
Opps. this is a typo. Thank you! Fixed